### DISCUSSION TITLE
Dubious virtual asset transactions

### DISCUSSION THREAD

**Discussion by yaksetig** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#35] [General]
Given the amendment's focus on enhancing the regulatory framework in alignment with Financial Action Task Force recommendations, is there a plan to introduce measures to strengthen the oversight of virtual asset transactions that might be used for money laundering or terrorism financing?

**Comment by FinalTwist** [comment]
I'm not sure I understand what you mean - could you give an example of what you mean by "measures to strengthen the oversight of virtual asset transactions" which wouldn't already be covered by the Act?

**Comment by CaraProvenance** [comment]
@yaksetig not sure if you are querying oversight by licensees or by CIMA.  If the former, I would note that the Anti-Money Laundering Regulations were updated, for the inclusion of Virtual Assets, in 2020. Updates to the sector specific guidance were subsequently published, these included virtual asset definitions, red flags and other FATF recommended VA considerations.  The regulations and guidance outline the specific obligations related to AML/CFT/CPF.  

Licensees are expected to build AML Compliance frameworks to comply with these AML regulations and as pertains to virtual asset transactions specifically, build control frameworks to mitigate the risks of money laundering, which as with fiat transactions, do exist. 

One element that I would suggest requires further guidance and solutions discovery for VASPs is compliance with the Travel Rule.
